api/src/test/java/org/openmrs/validator/ConceptValidatorTest.java

```diff
@@ -368,11 +357,9 @@ public void validate_shouldFailIfThereIsADuplicateUnretiredConceptNameInTheSameL
 		anotherConcept.getFullySpecifiedName(en).setName(duplicateName);
 		Errors errors = new BindException(anotherConcept, "concept");
 		
-		expectedException.expect(DuplicateConceptNameException.class);
-		expectedException
-		        .expectMessage("'" + duplicateName + "' is a duplicate name in locale '" + en + "'");
 		validator.validate(concept, errors);
-		validator.validate(anotherConcept, errors);
```

Comment:
You need to remove the line `validator.validate(concept, errors);`
Since it is failing the test at https://travis-ci.org/github/openmrs/openmrs-core/jobs/702528088#L2893

I made the mistake some time back and added this line https://github.com/openmrs/openmrs-core/commit/e4cb346ee8c63915c1261e8412340b8d0eb6ae16#diff-1f3fc20c6e60d3bf5fcdda09c14456c2R374

The test is making sure that if you validate a concept with a duplicate name the validation fails. It will throw a `DuplicateConceptNameException` which it does on the call to `validate`.

The call `validator.validate(concept, errors);` throws this exception but since its not in the assertThrows it fails the test. The `() -> validator.validate(anotherConcept, errors)` in your assertThrows is testing exactly that and is enough for us to make sure `validate` works as expected in this case.